.github/workflows/test.yml

```
@@ -62,6 +62,7 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
       - name: Run tests with Thread Sanitizer
+        timeout-minutes: 15
```

```suggestion
        timeout-minutes: 10
```

Looks like all the other tests are succeeding in 4-5 mins. This should still give us wiggle room but also unclog the queue a bit more.